lib/ex_aws/timestream/write/measure_value.ex

```diff
@@ -0,0 +1,23 @@
+defmodule ExAws.Timestream.Write.MeasureValue do
+  alias __MODULE__
+
+  @moduledoc """
+  Measure Value for the time-series data point.
+  https://docs.aws.amazon.com/timestream/latest/developerguide/API_MeasureValue.html
+  """
+
+  defstruct name: [],
+            value: nil,
+            type: nil
+
+  @type measure_value :: %MeasureValue{}
+
+  @doc "Create a new MeasureValue struct"
+  @type new_opts :: %{
+    name: binary,
+    value: binary,
+    type: binary,
+  }
+  @spec new(name :: binary, value :: binary, type :: binary) :: %MeasureValue{}
```

Comment:
Given that %MeasureValue{} has been defined as the type `measure_value` just above, it seems like maybe that should be used here?

Reply:
👍